pkg/container/network.go

```diff
@@ -31,14 +35,34 @@ var ignoreInterfaces = map[string]struct{}{
 	"lo": {},
 }
 
-func SetupContainerNetworking() ([]DHCPInterface, error) {
+var (
+	maxIntfsVar     = "IGNITE_INTFS"
+	defaultMaxIntfs = 1
+)
+
+type FCInterface struct {
+	VMTAP     string
+	MACFilter string
+}
+
+func SetupContainerNetworking(args map[string]string) ([]FCInterface, []DHCPInterface, error) {
```

Comment:
Here instead of `map[string]string` containing custom-parse env, we can just parse annotations.
Alternatively, you could pass the entire VM object.
Ultimately, we will have a specific `VM.Spec.NetworkInfo` sub-type that could be passed here should the annotations graduate into a future API version.